src/tests/mocks/ownable_mocks.cairo

```diff
@@ -1,22 +1,22 @@
 #[starknet::contract]
 mod DualCaseOwnableMock {
     use openzeppelin::access::ownable::OwnableComponent;
-    use openzeppelin::access::ownable::mixins::OwnableDual;
+    use openzeppelin::access::ownable::mixins::OwnableMixin;
     use starknet::ContractAddress;
 
     component!(path: OwnableComponent, storage: ownable, event: OwnableEvent);
-    component!(path: OwnableDual, storage: ownabledual, event: OwnableDualEvent);
+    component!(path: OwnableMixin, storage: ownablemixin, event: OwnableMixinEvent);
```

Comment:
I understand why this is required, but having these two components as dependencies seems weird. I would favor having the Mixins impl inside the OwnableComponent to avoid this, but then the components would grow. I'm leaning toward the second approach if we have to chose.

Reply:
> I understand why this is required, but having these two components as dependencies seems weird. I would favor having the Mixins impl inside the OwnableComponent to avoid this, but then the components would grow. I'm leaning toward the second approach if we have to chose.

Indeed, this was the workaround to not bloating the main component. I'll update to the fully-loaded mixin within the component and see what we think